turtle_following_node:
  ros__parameters:
    turtle2_init_pose_x: 0.0
    turtle2_init_pose_y: 0.0
    turtle2_init_pose_theta: 0.0
    kp_linear: 1.5
    kp_angular: 6.0
